packages/admin/cms-admin/src/dam/FileForm/FileSettingsFields.tsx

```diff
@@ -115,6 +115,13 @@ export const FileSettingsFields = ({ isImage, folderId }: SettingsFormProps): Re
                         name="license.type"
                         label={<FormattedMessage id="comet.dam.file.type" defaultMessage="Type" />}
                         fullWidth
+                        required={damConfig.requireLicense}
+                        validate={(value: string) => {
+                            if (damConfig.requireLicense && value === "NO_LICENSE") {
+                                return <FormattedMessage id="comet.dam.file.error.license.type" defaultMessage="License type is required" />;
+                            }
+                        }}
+                        shouldShowError={() => true}
```

Comment:
Why do you need this? This should be the default

Reply:
this was a suggestion by Thomas in the previous comments

Reply:
@thomasdax98 doesn't this show the error even when the user hasn't touched the form yet? 🤔